index.js

```diff
@@ -258,6 +266,18 @@ module.exports = {
     return this._coveredAddon;
   },
 
+  /**
+   * Find the module (if any) that's being covered.
+   * @returns {Module} the module under test
+   */
+  _findCoveredModule: function() {
```

Comment:
I don't understand why we need this (we should use `_findCoveredAddon` directly)